src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Dropdown from "react-dropdown";
+import styles from "./Dropdown.scss";
+
+const options = ["szkola1", "szkola2", "szkola3"];
```

Comment:
`options` for Dropdown should be passed using `constructor`, not as a constant array inside component module.